docs/utilities/idempotency.md

```diff
@@ -884,6 +960,49 @@ When creating the `DynamoDBPersistenceStore`, you can set a custom [`DynamoDbCli
         .build();
     ```
 
+### Customizing Redis client
+
+The `RedisPersistenceStore` uses the [`JedisPooled`](https://www.javadoc.io/doc/redis.clients/jedis/latest/redis/clients/jedis/JedisPooled.html) java client to connect to the Redis standalone server or the [`JedisCluster`](https://javadoc.io/doc/redis.clients/jedis/4.0.0/redis/clients/jedis/JedisCluster.html) to connect to the Redis cluster.
+When creating the `RedisPersistenceStore`, you can set a custom Jedis client:
+
+=== "Custom JedisPooled with connection timeout"
+
+    ```java hl_lines="2-11 13 18"
+    public App() {
+        JedisConfig jedisConfig = JedisConfig.Builder.builder()
+            .withHost(redisCluster.getHost())
```

Comment:
where does `redisCluster` come from? Would be great to have the full detail here as it's not super easy...

Reply:
c&p from code. I will replace that